docs/getting_started.md

```diff
@@ -30,62 +32,105 @@ If you see any changes between the two feel free to create a
 [pull request](https://github.com/google/flax/compare)
```

Comment:
This is no longer true as we've heavily modified the notebook. Maybe we should remove this note?